vyper/premade_contracts.py

```diff
@@ -2,8 +2,8 @@
 
 erc20 = """
 class ERC20():
-    def name() -> bytes32: constant
-    def symbol() -> bytes32: constant
+    def name() -> bytes[40]: constant
+    def symbol() -> bytes[5]: constant
```

Comment:
Because of the way bytearray works we could just make them `bytes[64]` and `bytes[32]` it occupies the same space, but is more flexible?

Reply:
Sounds nice 👍

Reply:
cc @fubuloubu

Reply:
WFM